api/controllers/volunteerForm.js

```diff
@@ -0,0 +1,95 @@
+const mongoose = require('mongoose');
+const { VolunteerForm, validateVolunteerForm, validateVolunteerFormLight } = require('../models/volunteerForm');
+
+exports.VolunteerFormGetAll = async (req, res, next) => {
+    const volunteerForms = await VolunteerForm.find();
+    res.send(volunteerForms);
+}
+
+exports.VolunteerFormGetOne = async (req, res, next) => {
+    if(!mongoose.Types.ObjectId.isValid(req.params.id))
+        return res.status(400).send('Podano błędny numer _id');
+    const volunteerForm = await VolunteerForm.findById(req.params.id);
+    if(!volunteerForm) return res.status(404).send('Formularz wolontariusza o podanym ID nie istnieje.');
+
+    res.send({
+        volunteerForm: volunteerForm,
+        request: {
+            type: 'GET',
+            description: 'Get all volunteer forms',
+            url: 'localhost:3000/api/volunteerForms/',
```

Comment:
czy przy deploymencie url podany w taki sposób nie spowoduje błędu? Nie jestem pewien czy tu powinno być localhost,
no i jesli aplikacja będzie np. na herokoku, to port może być inny niż 3000.

Reply:
Wydaje mi się, że po prostu po deploymencie to zaktualizuje i powinno być okej